modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,173 @@ RETURN ltrim('   hello')
 
 ======
 
+
+
+[[functions-normalize]]
+== normalize()
+
+`normalize()` returns the given `STRING` normalized using the `NFC` Unicode normalization form.
+
+[NOTE]
+====
+Unicode normalization is a process that transforms different representations of the same string into a standardized form.
+For more information, see the documentation for link:https://unicode.org/reports/tr15/#Norm_Forms[Unicode normalization forms].
+====
+
+The `normalize()` function is useful for converting `STRING` values into comparable forms.
+When comparing two `STRING` values, it is their Unicode codepoints that are compared.
+In Unicode, a codepoint for a character that looks the same may be represented by two, or more, different codepoints.
+For example, the character `<` can be represented as `\uFE64` (﹤) or `\u003C` (<).
+To the human eye, the characters may appear identical.
+However,  if compared, Cypher will return false as `\uFE64` does not equal `\u003C`.
```

Comment:
"Cypher" appears as "Cypher (r)" here but not on other pages. I find this a bit inconsistent although I don't think that's a problem Gem could fix...

cc. @JPryce-Aklundh

Reply:
The first time that Cypher is mentioned on each page, we have to add that to show its status as a registered trademark. A way to fix it would be to mention Cypher earlier in the page. Will have a look.